clean up

jwatzek opened the pull request:
- Turn into R notebook, add section headers
- Add helper function for data frame checks to reduce redundancy
- Clean up (some) formatting for consistency

I made sure this creates the same file as before (for `norm_age`, this is "just" up to ~10 digits after the decimal). I did run into 2 conceptual issues/questions:

1. DPZ ring-tailed lemur Amb, trial 8. `pick = 9`. Not sure if that was supposed to be NA or one of (1, 2, 3). If the former, then exclude (not currently the case); if the latter, correct this.
1. At Lincoln Park Zoo, `board_size = 35.5` in the raw file, but is changed to `board_size = 61.5`. Please add a note on where this new number came from.

jwatzek committed:
clean up

- Turn into R notebook, add section headers
- Add helper function for data frame checks to reduce redundancy
- Clean up (some) formatting for consistency

jwatzek committed:
remove

jwatzek commented:
@manuelbohn Just saw that the board size was changed in commit 2849e9aa2a6fa70f77e2e9fe6023d0fdf52c6c86. For reproducibility, it would still be good to document where this came from 👍

manuelbohn commented:
@jwatzek Board size was submitted in inch and I changed it to cm. I made a comment in the commit, but maybe there is a better place to put this info?

manuelbohn commented on analysis/01_data_processing.Rmd:
````diff
@@ -1,703 +1,404 @@
+         board_size = 61.5)
 ```
 
+> **!!!** Where does the new board_size number (61.5) come from? --jw
````
convert inch to cm

jwatzek replied:
@manuelbohn So that's what I thought at first but 35.5 inches = 90 cm...

Re: where to put it, I'll make a comment directly in the code before merging (same as for the ACCI board size)

manuelbohn replied:
Ok, my memory really did not serve me well here. I went through my emails again and found an exchange with Lydia in which I asked her if the board size was correct (35 cm seemed to small given the cup distance). She replied that 35 cm was the depth and not the width of the board. the 61.5cm is the value she gave me for the width. Sorry for the confusion!

manuelbohn commented:
For the 9 in in the dpz lemur pick data - not sure what it means. I'd exclude the trial.

jwatzek committed:
edit data checks

- Exclude DZP Lemur trial in which `pick = 9`
- Fix data check to look for columns with `any` (not `all`) NAs
- Add data check on merged data on whether `correct` value matches
  whether `hiding_location` and `pick` are identical

Some lines fail the last check -- needs action

jwatzek commented:
I excluded the lemur trial. I also added another data check for whether `correct` matches whether `hiding_location` and `pick` are identical and there are a few cases where this fails (the table below is from the .nb.html file). What do we do here? Overwrite the `correct` col?

<img width="824" alt="screen shot 2018-10-08 at 6 44 44 am" src="https://user-images.githubusercontent.com/6778316/46589314-da9eac00-cac5-11e8-86ca-06a7c59c107d.png">

manuelbohn commented:
Yes, let's override the the correct col.

jwatzek committed:
clean data

- Resolve Lincoln Park board_size issue (add comment)
- Overwrite correct column to match whether hiding_location and
  pick are identical

jwatzek commented:
Shipped 🚢
